infrastructure/modules/worker/terraform.tf

```diff
@@ -0,0 +1,79 @@
+module "service" {
+  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/service?ref=v2.4.1"
+
+  task_definition_arn            = module.task_definition.arn
+  service_name                   = var.name
+  cluster_arn                    = var.cluster_arn
+  subnets                        = var.subnets
+  service_discovery_namespace_id = var.namespace_id
+  launch_type                    = "FARGATE"
+  desired_task_count             = var.desired_task_count
+}
+
+module "autoscaling" {
+  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/autoscaling?ref=v2.4.1"
+
+  name = var.name
+
+  cluster_name = var.cluster_name
+  service_name = var.name
+
+  min_capacity = var.min_capacity
+  max_capacity = var.max_capacity
+}
```

Comment:
Does the worker need to export the scale_high/scale_low actions in `outputs.tf`? Otherwise how can you attach this to a queue?

Reply:
Ah ok. Am I right that we would need to attach the scaling policy to a queue for `desired_task_count = 0` to work? Otherwise I don't see how it would know work has to be performed and scale up.

Reply:
Yeah, they need to be attached to queues to do anything useful.